extensions/proposals/EXT_multi_draw_arrays/extension.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<proposal href="proposals/EXT_multi_draw_arrays/">
+  <name>EXT_multi_draw_arrays</name>
+
+  <contact> <a href="https://www.khronos.org/webgl/public-mailing-list/">WebGL
+  working group</a> (public_webgl 'at' khronos.org) </contact>
+  
+  <contributors>
+    <contributor>Contributors to the EXT_multi_draw_arrays specification</contributor>
+    <contributor>Members of the WebGL working group</contributor>
+  </contributors>
+
+  <number>NN</number>
+
+  <depends>
+    <api version="1.0"/>
+  </depends>
+
+  <overview>
+    <mirrors href="https://www.khronos.org/registry/OpenGL/extensions/EXT/EXT_multi_draw_arrays.txt"
+             name="EXT_multi_draw_arrays">
+    </mirrors>
+	
+    <div class="nonnormative">
+      <p>This extension exposes the EXT_multi_draw_arrays functionality to WebGL.</p>
+
+      <p>CAD vendors rendering large models comprised of many individual parts face scalability issues issuing large numbers of draw calls from WebGL. This extension reduces draw call overhead by allowing better batching.</p>
+    </div>
+
+    <features>
+      <feature>The <code>multiDrawArraysEXT</code> and <code>multiDrawElementsEXT</code> entry points are added. These provide a counterpoint to instanced rendering and are more flexible for certain scenarios.</feature>
+
+      <feature>The <code>offset</code> arguments to <code>multiDrawArraysEXT</code> and <code>multiDrawElementsEXT</code> choose the starting offset into their respective typed arrays or sequences. This primarily avoids allocation of temporary typed array views.</feature>
+    </features>
+  </overview>
+
+  <idl xml:space="preserve">
+[NoInterfaceObject]
+interface EXT_multi_draw_arrays  {
+  void multiDrawArraysEXT(GLenum mode, 
+                          (Int32Array or sequence&lt;GLint&gt;) firstList, GLuint firstOffset,
```

Comment:
Consider `firstsList`, `firstsOffset`, `countsList`, `countsOffset`, which I think is clearer about them being lists of firsts and counts. `firstOffset` and `countOffset` in particular give the wrong impression.

Reply:
Thanks, good idea - renamed.